packages/syft/setup.cfg

```diff
@@ -164,7 +164,7 @@ ci-grid =
     flask_sockets
     flask_sqlalchemy
     names
-    PyInquirer
+;      PyInquirer # commenting out until we sort out prompt-toolkit dep conflict with Jupyter notebook
```

Comment:
ci-grid is used only in CI, this shouldn't be commented out.

Reply:
fixed